functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -199,36 +259,61 @@ Then it will export the results to Export-DbaDiagnosticQuery.
                 $QueryName = Invoke-DiagnosticQuerySelectionHelper $parsedscript
                 $first = $false
             }
+            #since some database level queries can take longer (such as fragmentation) calculate progress with database specific queries * count of databases to run against into context
+            $CountOfDatabases = ($databases).count
+
 
+            if ($QueryName.Count -ne 0) {
+                #if running all queries, then calculate total to run by instance queries count + (db specific count * databases to run each against)
+                $CountOfDatabaseSpecific = @($parsedscript | Where-Object {$_.QueryName -in $QueryName -and $_.DBSpecific -eq $true}).count
```

Comment:
We use camel casing for variables that are not part of the parameter block of the function... plus you could also use shorthand for database references (e.g. `$countDbSpecific`).